src/Riok.Mapperly/Emit/SyntaxFactoryHelper.cs

```diff
@@ -299,6 +299,14 @@ public static InvocationExpressionSyntax StaticInvocation(string receiverType, s
         return InvocationExpression(methodAccess).WithArgumentList(ArgumentList(arguments));
     }
 
+    public static string StaticMethodString(IMethodSymbol method)
+    {
+        var qualified =
+            FullyQualifiedIdentifierName(method.ReceiverType?.NonNullable()!)
+            ?? throw new ArgumentNullException(nameof(method.ReceiverType));
```

Comment:
I don't think this works as expected, it looks like we have the same problem at other locations in this file.
As far as I understand it what tried achieving here would be sth. like this:
```c#
var receiver = method.ReceiverType ?? throw new NullReferenceException(nameof(method.ReceiverType) + " is null");
var qualifiedReceiverName = FullyQualifiedIdentifierName(receiver.NonNullable());
```

Reply:
Updated for `StaticInvocation` and `StaticMethodString` methods